gulpfile.js

```diff
@@ -97,6 +97,11 @@ const traverseJson = (jsonTree, descriptionCallback, prefixPath) => {
     }
 };
 
+// remove invalid characters that aren't allowed in xlf.
+const removeInvalidXLFCharacters = (str) => {
+    return str.replace(/[^a-zA-Z_]/g, "");
```

Comment:
Why are these invalid characters?

Reply:
They are invalid according to the tool that does the localization, the OneLocBuild task. I didn't find any details on why